packages/passport/sdk/src/storage/IndexedDbStorage.ts

```diff
@@ -0,0 +1,173 @@
+import { AsyncStorage } from 'oidc-client-ts';
+
+const databaseName = 'ImmutableSDK';
+const storeName = 'Passport';
+
+export class IndexedDbStorage implements AsyncStorage {
```

Comment:
Given that this class is used for maintaining user state, it's pretty crucial that this continues to work as expected. Can we please add some tests?

Reply:
It just uses `localForage` now. Do we need tests?